WordPress/src/main/java/org/wordpress/android/ui/compose/unit/Margin.kt

```diff
@@ -0,0 +1,12 @@
+package org.wordpress.android.ui.compose.unit
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Suppress("MagicNumber")
+sealed class Margin(val value: Dp) {
+    object Small : Margin(4.dp)
+    object ExtraLarge : Margin(16.dp)
+    object Medium : Margin(8.dp)
+    object ExtraExtraMediumLarge : Margin(32.dp)
+}
```

Comment:
Nice 💯

Reply:
We have a similar [idea](https://github.com/woocommerce/woocommerce-android/pull/6524#issuecomment-1128593595), but we still didn't implement it 🤦 😅, the different is that we wanted just to have a single `object` with functions:
```kotlin
object Dimens {
    @Composable
    fun medium() = dimensionResource(id = R.dimen.major_100)
    ...
}

@Composable
fun Screen() {
    ....padding(Dimens.major100())
}
```

This will allow us to reuse the same dimensions we already have for other screens.

Reply:
That's also a great approach!

Imho since we're just entering the Compose world I'd like to keep the option open of "redefining" the dimens we're using, in other words keep them separate from the XML ones.

Wdyt?

Reply:
I think it's a fine approach too, I just presented the other approach, but it wasn't meant as a replacement for your approach